modules/openapi-generator/src/main/resources/php-symfony/composer.mustache

```diff
@@ -25,7 +25,7 @@
         "ext-mbstring": "*",
         "symfony/validator": "*",
         "jms/serializer-bundle": "*",
-        "symfony/framework-bundle": "^2.3|^3.0"
```

Comment:
Why not `^4.0`?

Reply:
I think we can go with `^4.1` to start with. If a user needs 4.0, he/she can do so by customizing the templates.